src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -261,22 +290,22 @@ def _get_node_maps(self, cores, gpus, threads_per_proc):
 
         return core_map, gpu_map
 
-
     # --------------------------------------------------------------------------
```

Comment:
pep-8.  The amount of white soace changes make the diff longer than needed - this method for example was not really changed at all.

Reply:
pep8ified all the DA related files